### PR TITLE
#15 Implement Web Driver Manager

### PR DESCRIPTION
**Pull request by konwinkler**
Resolve issue #15:
To no longer manually manage the web driver executables a
webdrivermanager library is utilized to download the drivers if
necessary. This is accomplished by calling the DriverManager.setup() in
Locomotive.java before starting the WebDriver.

### CODE REVIEW

**Comment by jossjacobo** [comment]
@konwinkler it looks good! Can you also remove all the `Heroku` code from the Chrome implementation? That is should've been removed a while ago. The chrome impl should work just like the other browsers.

**Comment by konwinkler** [comment]
@jossjacobo Done.

**Comment by konwinkler** [comment]
@jossjacobo Done

File: src/main/java/io/ddavison/conductor/Locomotive.java [hunk 1]
```diff
@@ -124,14 +107,7 @@ public Locomotive() {
...
-                    String herokuChromeDriver = JvmUtil.getJvmProperty("GOOGLE_CHROME_BIN");
-                    System.out.println("GOOGLE_CHROME_BIN: " + herokuChromeDriver);
-
-                    if (herokuChromeDriver != null && !herokuChromeDriver.isEmpty()) {
-                        System.setProperty("webdriver.chrome.driver", herokuChromeDriver);
-                    }
+                    ChromeDriverManager.getInstance().setup();
 
                     ChromeDriverService service = new ChromeDriverService.Builder()
                             .usingDriverExecutable(new File(JvmUtil.getJvmProperty("webdriver.chrome.driver")))
```
Comment (jossjacobo) [inline src/main/java/io/ddavison/conductor/Locomotive.java]: It shouldn't affect anything but this `builder` stuff can go too, it was also part of the heroku thing.

**Comment by jossjacobo** [comment]
👍

**Pull request merged**